hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -1,131 +1,110 @@
+If `1` and `WAKE_ON_GPIO=0` then `sudo halt` will switch off all PMIC outputs. This is lowest possible power state for halt but may cause problems with some HATs because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.
 
-Default: 0  
+Default: `0`  
 Version: 2019-07-15  
 
 ### BOOT_ORDER
-The BOOT_ORDER setting allows flexible configuration for the priority of different bootmodes. It is represented as 32bit unsigned integer where each nibble represents a bootmode. The bootmodes are attempted in lowest significant nibble to highest significant nibble order.
-
-E.g. 0x21 means try SD first followed by network boot then stop. Whereas 0x2 would mean try network boot and then stop without trying to boot from the SD card.
+The `BOOT_ORDER` setting allows flexible configuration for the priority of different bootmodes. It is represented as 32bit unsigned integer where each nibble represents a bootmode. The bootmodes are attempted in lowest significant nibble to highest significant nibble order.
 
+E.g.`0xf41` means continuously trying SD card followed by USB mass storage. Whereas, `0xf12` means continuously trying network boot followed by SD card boot. 
 The retry counters are reset when switching to the next boot mode.
 
-BOOT_ORDER fields  
+`BOOT_ORDER` fields  
 The BOOT_ORDER property defines the sequence for the different boot modes. It is read right to left and up to 8 digits may be defined.
 
-* 0x0 - NONE (stop with error pattern)
-* 0x1 - SD CARD
-* 0x2 - NETWORK
-* 0x3 - USB device boot - Reserved - Compute Module only.
-* 0x4 - USB mass storage boot (since 2020-09-03)
-* 0xf - RESTART (loop) - start again with the first boot order field. (since 2020-09-03)
+| Value | Mode      | Description                                                                                       |
+|-------|-----------|---------------------------------------------------------------------------------------------------|
+|  0x1  |  SD CARD  |  SD card (or EMMC on Compute Module 4)                                                            |
+|  0x2  |  NETWORK  |  Network boot                                                                                     |
+|  0x3  |  USB DEV  |  USB device boot - See [usbboot](https://github.com/raspberrypi/usbboot)  (since 2020-09-03)      |
+|  0x4  |  USB MSD  |  USB mass storage boot (since 2020-09-03)                                                         |
+|  0xe  |  STOP     |  Stop and display error pattern (since 2020-09-03). A power cycle is required to exit this state. |
+|  0xf  |  RESTART  |  Start again with the first boot order field. (since 2020-09-03)                                  |
+
+After trying each non-zero boot-mode the bootloader stops. However, from 2020-09-03 the bootloader will monitor the SD card-detect pin and try SD boot if a new SD card is inserted.
```

Comment:
`boot-mode` -> `boot mode`

Reply:
`card-detect` -> `card detect`

Reply:
card-detect is a specific pin on the SD slot, so should possibly remain hyphenated.